client/verta/docs/tutorials/deployment/endpoint_config.rst

```diff
@@ -0,0 +1,89 @@
+Endpoint configuration
```

Comment:
This tutorial will become massive, with sub sub sections for everything that is configurable. It seems less like a tutorial and more like a section-level thing, right?

Reply:
>It seems less like a tutorial and more like a section-level thing, right?
I'm trouble understanding what "section-level thing" means.

Would you suggest making each of these subsubsections (autoscaling, resources) its own page? If so, that makes sense to me.

Reply:
Yes. We'll soon add more and more knobs.

Reply:
I've split endpoint_config.rst into knob-focused tutorials, and then deleted endpoint_config.rst because all that remained of it was "`endpoint.update()` exists! See other tutorials." which imo is redundant and doesn't address a particular problem